### PR TITLE
My first Microverse project

### PR DESCRIPTION
**Pull request by lawrahkonwea**
This is my first Microverse project, and I added the following;

- markup file (html) added
- cascading style sheet (css) added
- node_module and test.md file which was ignored by the gitignore file
- descriptive readme file added
- linters error fixed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,97 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (fobadara) [inline README.md:1]: [REQUIRED] Kindly make your README is more descriptive. Use the [Microverse template](https://github.com/microverseinc/readme-template)

**Review by fobadara — CHANGES_REQUESTED** [review]
# REQUIRES CHANGES ♻️ 
Hi @lawrahkonwea, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing

## Required Changes ♻️
_Check the comments under the review._
I had to include some suggestions here because there is no way to use inline comments.
- [ ] Kindly name your branches after a feature or action like `hello_world` or `add_hello_world`. You can read this [article](https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/rename-Git-branch-local-remote-GitHub-GitLab-BitBucket#:~:text=The%20steps%20to%20change%20a,name%20from%20your%20remote%20repo) n how to change the name.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- Your commit messages should be in the imperative. Like `Fix` instead of `Fixing`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lawrahkonwea** [comment]
Hello Fobadara,
I have renamed the branch and deleted the previous branch, it is prompting
a new pull request. From the earlier instructions, it was stated that a new
pull request should not be opened unless instructed. Please, I need help
with what to do. Thank you.

**Comment by lawrahkonwea** [comment]
Hello Fobadara,
I have renamed the branch and deleted the previous branch, it is prompting
a new pull request. From the earlier instructions, it was stated tht a new
pull request should not be opened unless instructed. Please I need a advice
on what to do. Thank you.

**Pull request closed without merging**